Python/cp/max_sum.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+
+Result = namedtuple("Result", "maxSum leftBound rightBound upBound lowBound")
+
+KadanesResult = namedtuple("KadanesResult", "maxSum start end")
+
+
+def kadanes(temp):
+    max = 0
+    maxStart = -1
+    maxEnd = -1
+    currentStart = 0
+    maxSoFar = 0
+
+    for i in range(0, len(temp)):
+        maxSoFar += temp[i]
+
+        if maxSoFar < 0:
+            maxSoFar = 0
+            currentStart = i + 1
+
+        if maxSoFar > max:
+            maxStart = currentStart
+            maxEnd = i
+            max = maxSoFar
+
+    return KadanesResult(max, maxStart, maxEnd)
+
+
+def max_sub_sub_rectangle(rectangle):
+
+    rows = len(rectangle)
+    cols = len(rectangle[0])
+
+    result = Result(float("-inf"), -1, -1, -1, -1)
+
+    for left in range(cols):
+        temp = [0 for _ in range(rows)]
+        for right in range(left, cols):
+            for i in range(rows):
+                temp[i] += rectangle[i][right]
+
+            res = kadanes(temp)
+            if result.maxSum > result.maxSum:
+                result = Result(res.maxSum, left, right, res.start, res.end)
+
+    return result
+
+if __name__ == '__main__':
```

Comment:
dynamic input

Reply:
@plazzy99  i did try it with dynamic input but it was taking a lot of time so I had to hard-code it

Reply:
that is not an issue 
go with dynamic input